client/noop.go

```diff
@@ -87,3 +92,8 @@ func (c *Noop) SetHTTPRequestSection(section string) Client {
 func (c *Noop) ResetHTTPRequestSection() Client {
 	return c
 }
+
+// Handler returns metrics endpoint for prometheus backend
+func (c *Noop) Handler() http.Handler {
+	return nil
```

Comment:
we'd better return handler always - in this case handler that, for example, returns `405` status code